inc/image.php

```diff
@@ -107,13 +109,17 @@ public function get_url( $params = [] ) {
 		if ( isset( $this->resize->get()['type'] ) ) {
 			$path_parts[] = $this->resize->toString();
 		}
-
+		// format is not a must that's what we are changing
```

Comment:
not sure what means this comment here ?

Reply:
some old comment from when i was modifying code to remember what i do, it means nothing for others, i will delete it